modules/core/shared/src/main/scala/scalacache/CacheAlg.scala

```diff
@@ -7,74 +7,64 @@ import scala.language.higherKinds
 /**
   * Abstract algebra describing the operations a cache can perform
   *
+  * @tparam F the effect of the cache. //todo elaborate
```

Comment:
I don't know how much elaboration is needed here. I think going too much more into depth on what `F[_]` means could be out of scope here.

Reply:
Yeah I think we can just remove the `@tparam` line completely. We'll assume that anyone using the library is familiar with tagless-final.